lib/webhook_system/event_log.rb

```diff
@@ -14,26 +14,34 @@ class EventLog < ActiveRecord::Base
     serialize :request, JSON
     serialize :response, JSON
 
-    def self.construct(subscription, event, request, response)
-      request_info = {
-        'event' => event,
-        'headers' => request.headers.to_hash,
-        'body' => request.body,
-        'url' => request.path,
-      }
-      response_info = {
-        'headers' => response.headers.to_hash,
-        'body' => response.body,
-      }
-
-      attributes = {
-        event_name: event['event_name'],
-        event_id: event['event_id'],
-        status: response.status,
-        request: request_info,
-        response: response_info,
-      }
-      subscription.event_logs.build(attributes)
+    class << self
+      def construct(subscription, event, request, response)
+        request_info = {
+          'event' => event,
+          'headers' => request.headers.to_hash,
+          'body' => truncated_string(request.body),
+          'url' => request.path,
+        }
+        response_info = {
+          'headers' => response.headers.to_hash,
+          'body' => truncated_string(response.body),
+        }
+
+        attributes = {
+          event_name: event['event_name'],
+          event_id: event['event_id'],
+          status: response.status,
+          request: request_info,
+          response: response_info,
+        }
+        subscription.event_logs.build(attributes)
+      end
+
+      private
+
+      def truncated_string(string)
```

Comment:
This gem already uses activesupport so take advantage of it .. 
http://api.rubyonrails.org/classes/String.html#method-i-truncate